content/en/guides/models/track/log/distributed-training.md

```diff
@@ -100,6 +100,52 @@ Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-tha
 
 The preceding image demonstrates the W&B App UI dashboard. On the sidebar we see two experiments. One labeled 'null' and a second (bound by a yellow box) called 'DPP'. If you expand the group (select the Group dropdown) you will see the W&B Runs that are associated to that experiment.
 
+### Organize distributed runs with `job_type`
+
+When running distributed training with multiple processes, you can specify the `job_type` in your [`wandb_init()`]({{< relref "/ref/python/sdk/functions/init/" >}}) invocation to distinguish between different types of nodes. Then you can create saved views to help you organize your runs and filter out noise from worker nodes.
+
+To organize your distributed training runs:
+
+1. Set `job_type` for different node types to categorize your nodes based on their function:
+
+   ```python
+   # Main coordinating node
+   wandb.init(
+       project="distributed-training",
+       group="experiment_1",
+       job_type="main"
+   )
+
+   # Reporting worker nodes
+   wandb.init(
+       project="distributed-training", 
+       group="experiment_1",
+       job_type="worker"
+   )
+   ```
+1. Create [saved views]({{< relref "/guides/models/track/workspaces/#create-a-new-saved-workspace-view" >}}) in your workspace to organize your runs. First, filter and group your runs to show a subset of runs, then click the **...** action menu at the top right and click **Save as new view**. For example, you could create the following saved views:
```

Comment:
You don't need to set `group` in `wandb.init()` to filter runs. If you have `job_type=""` you can go to your **Runs** tab and filter with **job_type**

Reply:
<img width="1101" height="500" alt="Screenshot 2025-08-26 at 2 33 53 PM" src="https://github.com/user-attachments/assets/4a79ef53-3bb3-42ef-a86f-a27aeb34bb96" />

Reply:
I struggled here because what you are filtering on in the UI is "Job Type", not `job_type`. With that drift, I struggled to figure out how best to describe the filter. Do you think it's OK to stick with the SDK variant?